### ISSUE TITLE
Load all defines

### ISSUE THREAD

**Issue by Alex9779** [tohara/OctoPrint-BigBoxFirmware#6]
Taking #2 to the next level it would be great to have a button to load all available defines, their values and enabled state. That would make it more easy the create a profile from the scratch. Just load everything then decide which to keep and which to change...

**Comment by tohara** [comment]
I actually started with that, but there is 500+ defines and you normally only need a few of the so it was almost more work to delete all I did not want than adding what I wanted. But sure, if it is really wanted there shouldn't any problem adding a function for it.

What i have been looking at is either adding autocomplete or drop down menu with all available defines instead of having to type it. But haven't figured out exactly how to do it yet, using builtin html5 functionality will add 500+ lines of html code for every define added, I did not like the delay from this when opening the profile dialog. (And it would not work on safari ;))
